### DISCUSSION TITLE
how to reconfigure? can't find .actrc file anywhere

### DISCUSSION THREAD

**Discussion by rob-zeno** [nektos/act#2401] [Q&A]
I know that the first time I ran "act", it asked me which size image I wanted, and I chose "Medium".  But now it turns out I've changed my mind.

However, I cannot find the ".actrc" file ANYWHERE, and I can't figure out how to get "act" to ask me again?

**Comment by rob-zeno** [comment]
Figured this out.  Had to read the XDG spec (slightly annoying) to find out that the file is NOT called ".actrc" but is stored in "~/.config/act/actrc").  Good now.

**Comment by ashishb** [comment]
@rob-zeno is that on Mac OS? I don't even have `~/.config/act/` directory.

**Reply by rob-zeno** [reply]
No, this was on a pretty new install of Ubuntu (although it is running in a VM on an ARM-based Mac).

**Comment by prestonbourne** [comment]
Did anyone figure this out for macos?

**Reply by jazzmano** [reply]
I have the samme issue on mac. Did you figure out where it resides?

**Reply by cditcher** [reply]
I found it in ~/Library/Application Support/act
